test/controllers/remote-controller.test.js

```diff
@@ -49,6 +51,29 @@ describe('RemoteController', function() {
     );
   }
 
+  it('increments a counter when onCreatePr is called', async function() {
+    const wrapper = shallow(createApp());
+    sinon.stub(shell, 'openExternal').resolves();
```

Comment:
Ah, this is the bit where you were having trouble getting the Promise to resolve?

Because the real `shell.openExternal` doesn't return a Promise, you might have better luck with:

```js
sinon.stub(shell, 'openExternal').callsArg(2);
```

And to report an error:

```js
sinon.stub(shell, 'openExternal').callsArgWith(2, new Error('boom'));
```

Reply:
ah thanks, I'll try that!